src/patient-search/patient-search.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { doSearch } from "./patient.resource";
-import { Link } from "react-router-dom";
```

Comment:
Using Link created a CSS nightmare, made it hard for the row to stretch across the whole table.